pkgs/applications/video/olive-editor/default.nix

```diff
@@ -1,34 +1,68 @@
-{ lib, stdenv, fetchFromGitHub
-, pkg-config, which, qmake, wrapQtAppsHook
-, qtmultimedia, frei0r, opencolorio_1, ffmpeg-full, CoreFoundation }:
+{
+  lib,
+  stdenv,
```

Comment:
I like giving each dependency its own line.

The convention to start each line with `, ` is much more common, do you have a strong reason to deviate from that here?

(`find . -name default.nix | while read line ; do head -2 $line | tail -1; done | grep "^  " | wc -l` is 289 while `find . -name default.nix | while read line ; do head -2 $line | tail -1; done | grep "^, " | wc -l` is 12885 ;) )

Reply:
I just formatted it with alejandra and didn't think about it much.